### DISCUSSION TITLE
What is the best way to use fontawesome with Maizzle?

### DISCUSSION THREAD

**Discussion by seyfer** [maizzle/framework#459] [Q&A] [answered]
That is a how-to question, which is not covered with Maizzle documentation.
Is there a way to integrate with Maizzle, or shall I just copy SVG from fontawesome and use it as an image?

My question is  - what is best for output?
* following this manual https://fontawesome.com/how-to-use/on-the-web/setup/using-package-managers and use full css `/css/all.css` and letting them be generated from `<i class="far far-user"></i>` - better for development
* use Icons as downloaded SVG images? - better for performance, harder to develop

**Comment by cossssmin** [comment, marked as answer]
- [`::before`](https://www.caniemail.com/features/css-pseudo-element-before/) and [`::after`](https://www.caniemail.com/features/css-pseudo-element-after/) have poor support, so using the classic FontAwesome class technique won't work
- SVG will not work in the majority of email clients due to security concerns - https://www.caniemail.com/features/image-svg/ 
 
My recommentation is to save SVGs as PNG and use them in `<img>` tags in your template, those will work everywhere.
